R/teal_slice.R

```diff
@@ -340,6 +336,37 @@ is.teal_slices <- function(x) { # nolint
 }
 
 
+# JSON utils ------------------------------------------------------------------------------------------------------
+
+slice_to_json <- function(x) {
+  if (!is.null(x$selected)) x$selected <- I(x$selected)
```

Comment:
Is it really needed? Do we really need `auto_unbox`? Scalar in R is a vector of length one anyway, so we don't need to make sure that it's saved as array.

Reply:
@gogonzo with the help of @chlebowa we made a huge simplification of it d5d353b229f99ce138887f3d1a0d52fd4b680a5b